src/main/features/core/windowTitleInfo.js

```diff
@@ -8,8 +8,14 @@ const updateDarwinTitle = (newTitle) => {
   }, 200);
 };
 
+// fix display of ampersands on windows
+const replaceAmpersands = string => (
+  process.platform === 'win32' ? string.replace('&', '&&&') : string
```

Comment:
Can you quickly explain why here it's triple "&" but I expected and in the other place it is just a double "&"?